src/Servers/Kestrel/test/InMemory.FunctionalTests/HttpsConnectionMiddlewareTests.cs

```diff
@@ -142,7 +142,7 @@ await using (var server = new TestServer(context => Task.CompletedTask, new Test
                 using (var connection = server.CreateConnection())
                 {
                     var stream = OpenSslStream(connection.Stream);
-                    await stream.AuthenticateAsClientAsync("localhost", new X509CertificateCollection(), SslProtocols.Tls12 | SslProtocols.Tls11, false);
+                    await stream.AuthenticateAsClientAsync("localhost", new X509CertificateCollection(), SslProtocols.None, false);
```

Comment:
Nit: This is now the [default behavior](https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslstream.authenticateasclientasync?view=netcore-3.1#System_Net_Security_SslStream_AuthenticateAsClientAsync_System_String_) for AuthenticateAsClientAsync so I think this is exactly equivalent to the following:

```suggestion
                    await stream.AuthenticateAsClientAsync("localhost");
```